### PR TITLE
MacOS quicker start

### PR DESCRIPTION
**Pull request by duncdrum**
see eXist-db/eXist#1274, eXist-db/eXist#1275, eXist-db/eXist#1248

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 build/
 expath-pkg.xml
+
+*.xpr
```
Comment (dizzzz) [inline .gitignore]: what program creates these files?

File: data/quickstart.xml [hunk 2]
```diff
@@ -39,9 +39,11 @@
                     on commercial Linux distributions. Users of other distributions are better
                     served by distribution-specific instructions. Instructions that eXist-db users
                     have found useful include: <ulink url="http://livingtao.blogspot.com/2012/01/few-easy-steps-to-install-sunoracle-jdk.html">Amazon Linux</ulink>, <ulink url="http://www.webupd8.org/2012/01/install-oracle-java-jdk-7-in-ubuntu-via.html">Ubuntu</ulink> and <ulink url="http://www.webupd8.org/2012/06/how-to-install-oracle-java-7-in-debian.html">Debian</ulink>.</para>
+                <para>macOS users: If you have installed Saxon as a JRE extension (via the extensions folder), this can cause a critical error during installation. To avoid this error user need to either
```
Comment (dizzzz) [inline data/quickstart.xml]: this is not only valid for macOS users, though we had recently two incidents on macOS only :-)

File: .gitignore (hunk 1 above)
Comment (duncdrum) [inline .gitignore]: Oxygen XML editor

Comment (joewiz) [inline .gitignore]: I would actually welcome an .xpr file. I've actually created one several times.

**Comment by joewiz** [comment]
I take it from @dizzzz's "Looks good" comment that he approves with his change.

**Pull request merged**